content/community/teams.md

```diff
@@ -0,0 +1,57 @@
+---
+prev: /community/
+next: /community/rules/
+---
+
+# Teams and Organization
+
+_This page is a stub._
+
+## Administrators
+
+Administrators are people who created, maintain, and are responsible for Codewars overall. They handle the most critical issues, like:
+- severe cases of violations of rules,
+- user accounts,
+- bugs,
+- ...?
+
+Cuttently, users with status of Administrator are:
+
+- [kazk](link to Codawars profile?)
+- ... others?
```

Comment:
jhoffner, alchemy, OverZealous. I think that's it or did I forget someone?

Reply:
Yeah I am just not sure if all of them are/want to be involved.

If you have any ideas on the topic, feel free to drop them in #98. Once discussion settles down, I will put ideas into this doc, un-draft the PR, and start sending review requests :)

Thanks!